Keda.Cosmosdb.Scaler/src/Services/CosmosDBLease.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Keda.CosmosDB.Scaler.Services
+{
+    public class CosmosDBLease
+    {
+        public string LeasesCosmosDBConnectionString { get; internal set; }
+        public string LeaseDatabaseName { get; internal set; }
+        public string LeaseCollectionName { get; internal set; }
+
+        public CosmosDBLease()
+        {
+        }
+
+        public CosmosDBLease(string leaseConnectionString, string leaseDatabaseName, string leaseCollectionName)
+        {
+            LeasesCosmosDBConnectionString = leaseConnectionString;

```

Comment:
add validation here for required params

Reply:
Do you want to throw regular exceptions or what do you want?

I'd use https://www.nuget.org/packages/Guard.NET/ here and use Guard.NotNullOrWhitespace() which is a convenient way imo